### PR TITLE
Update Rook version from 1.3.2 to 1.3.4

### PR DESCRIPTION
**Pull request by bipuladh**
Changes required to support the external cluster are only present in Rook 1.3.4.

### CODE REVIEW

File: deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1245,7 +1250,7 @@ spec:
                 - name: OPERATOR_NAME
                   value: ocs-operator
                 - name: ROOK_CEPH_IMAGE
-                  value: rook/ceph:v1.3.2
+                  value: rook/ceph:v1.3.4
```
Comment (travisn) [inline deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml:1253]: If you use this tag `rook/ceph:v1.3.4-4.gb97be10` for the image you will pick up the latest change for the default `data` field on the device sets.

Comment (bipuladh) [reply deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml]: ack.

**Pull request merged**